Public/Src/FrontEnd/Script/Ambients/Transformers/AmbientTransformer.Process.cs

```diff
@@ -437,6 +439,13 @@ private void ProcessExecuteArguments(Context context, ObjectLiteral obj, Process
 
             // TODO: Regex. Should we follow ECMA, C#, JavaScript?
 
+            // Weight.
+            var weight = Converter.ExtractInt(obj, m_weight);

```

Comment:
>ExtractInt [](start = 35, length = 10)

Why isn't Converter.ExtractOptionalInt() necessary here?

Reply:
Oh, maybe that's why the builds are failing :)

---
In reply to: [271087913](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271087913) [](ancestors = 271087913)